waflyctl.go

```diff
@@ -14,18 +14,20 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"github.com/BurntSushi/toml"
-	"github.com/sethvargo/go-fastly/fastly"
-	"gopkg.in/alecthomas/kingpin.v2"
-	"gopkg.in/resty.v1"
 	"io"
 	"io/ioutil"
 	"log"
 	"os"
 	"os/user"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/sethvargo/go-fastly/fastly"
+	"gopkg.in/alecthomas/kingpin.v2"
+	"gopkg.in/resty.v1"
 )
 
```

Comment:
Is this the result of gofmt?

Reply:
It's goimports, which by default runs on save in VS code.
Personally I much prefer this layout/ordering, but if you disapprove, I can disable it.

Reply:
I think sticking to gofmt makes it equally good or bad for everyone (and will avoid back and forth depending on the editor and customisations you use), so if you don't mind let's stick to it.

Reply:
This output passes straight through gofmt - so there should be no back & forth or issues.
goimports also removes unused dependancies etc. So I'd certainly recommend it. 
But I can disable it in my IDE if you like.

Reply:
You are right. The space is making the difference.